src/main/kotlin/no/nav/amt/arena/acl/processors/TiltakGjennomforingProcessor.kt

```diff
@@ -154,3 +153,5 @@ open class TiltakGjennomforingProcessor(
 	private fun ugyldigGjennomforing(data: ArenaTiltakGjennomforing) =
 		data.ARBGIV_ID_ARRANGOR == null || data.LOKALTNAVN == null
 }
+
+private fun UUID.digest() = DigestUtils.md5DigestAsHex(ObjectMapperFactory.get().writeValueAsString(this).toByteArray())
```

Comment:
Man trenger kanskje ikke ObjectMapper for å lage en digest av en UUID, det holder muligens med `this.toString()`

Reply:
Fixed